app/res/values-de/strings.xml

```diff
@@ -148,8 +148,8 @@
     <string name="gist_description_hint">erstellt mit Android</string>
     <string name="title">Titel</string>
     <string name="edit">Bearbeiten</string>
-    <string name="starring_gist">Füge Gist als Hervorhebung hinzu…</string>
-    <string name="unstarring_gist">Entferne hervorgehobenen Gist…</string>
+    <string name="starring_gist">Füge Gist als Favorit hinzu…</string>
+    <string name="unstarring_gist">Entferne favorisierten Gist…</string>
```

Comment:
This sounds like the gist will be removed instead of the gists starring status.

Something like "Entferne (den) Gist von (den) Favoriten" or "Defavorisiere (den) Gist" or "Entferne den Favoritenstatus des Gists" would be better